src/Benchotron/UI/Console.purs

```diff
@@ -94,9 +93,9 @@ runBenchmarkConsole benchmark = do
   lift $ do
     stderrWrite "\n"
     noteTime \t -> "Finished at: " <> t <> "\n"
-  return r
+  pure r
   where
-  noteTime f = now >>= toLocaleTimeString >>= (stderrWrite <<< f)
+  noteTime f = now >>= show >>= (stderrWrite <<< f)
```

Comment:
Yes, this is much better. Ideally we would not depend on `JSDate` at all, I think, but as far as I'm aware there's not yet a library for converting a `DateTime` into a locale string directly.